src/main/types/SIOP.types.ts

```diff
@@ -196,6 +198,11 @@ export enum ResponseContext {
   OP = 'op',
 }
 
+export enum PresentationExchangeContext {
```

Comment:
Suggest to make this something like PresentationExchangeVersion en then have NONE and V1_0 for instance. This allows us to respond to changes in the PE spec in the future